frida_tools/tracer.py

```diff
@@ -210,6 +214,11 @@ def include_debug_symbol(self, *function_name_globs):
             self._spec.append(('include', 'debug_symbol', f))
         return self
 
+    def init_session_paths(self, *function_name_globs):
+        for f in function_name_globs:
+            self._init_session_paths.append(f)
+        return self
+
```

Comment:
Dead code left-over from the earlier approach.

Reply:
Done.